### PR TITLE
web-apps(back): Change the string formatting

### LINKED ISSUE #5494: Python backend code for crud-web-apps should run with lower versions than Python 3.6
/kind feature

**Why you need this feature:**
Right now the Python backends and the common backend for our [crud-web-apps](https://github.com/kubeflow/kubeflow/tree/master/components/crud-web-apps) is using `f"strings"` in a lot of places. This requires an environment of Python 3.6 and up to be installed. A lot of linux installations come with a lower version of Python preinstalled, so it adds extra steps to new developers who might want to contribute code to the backend code.

**Describe the solution you'd like:**
Remove any use of `f"strings"` in the codebase and use the `"%s world" % var"` syntax for doing string substitutions.

**Anything else you would like to add:**
`f"string"` free code:
- [ ] [common backend](https://github.com/kubeflow/kubeflow/tree/master/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend)
- [ ] [jupyter web app](https://github.com/kubeflow/kubeflow/tree/master/components/crud-web-apps/jupyter/backend)

/assign kimwnasptd

### PR DESCRIPTION
**Pull request by kimwnasptd**
partially fixes #5494 

Removes the use of `f"strings"` from the common backend code.

/cc @elikatsis 
/cc @thesuperzapper 
/assign @kimwnasptd

### CODE REVIEW

**Review by elikatsis — COMMENTED** [review]
@kimwnasptd I added some nit comments. No strong opinion, just suggestions :smile: 

Also, would you mind adding the `subresource` argument changes in a separate commit so that the change is visible in the final commit message? (although they don't affect any code as of now)

**Comment by thesuperzapper** [comment]
/hold

I believe this change is not needed, see my comment: https://github.com/kubeflow/kubeflow/issues/5496#issuecomment-759056472

File: components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/api/utils.py [hunk 1]
```diff
@@ -26,4 +26,5 @@ def failed_response(msg, error_code):
 
 
 def events_field_selector(kind, name):
-    return f"involvedObject.kind={kind},involvedObject.name={name}"
+    selector = "involvedObject.kind=%s,involvedObject.name=%s" % (kind, name)
+    return selector
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/api/utils.py]: Done.

File: components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/authz.py [hunk 2]
```diff
@@ -79,25 +75,39 @@ def is_authorized(user, verb, namespace, group, version, resource):
...
+    if group == "":
+        msg += " %s/%s" % (version, resource)
+    else:
+        msg += " %s/%s/%s" % (group, version, resource)
+
+    if subresource is not None:
+        msg += "/%s" % subresource
+
+    if namespace is not None:
+        msg += " for namespace '%s'" % namespace
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/authz.py]: Done.

File: components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/authz.py [hunk 3]
```diff
@@ -79,25 +75,39 @@ def is_authorized(user, verb, namespace, group, version, resource):
         return False
 
 
-def ensure_authorized(verb, group, version, resource, namespace=None):
+def generate_unauthorized_message(verb, group, version, resource,
```
[the remark this answers is not in the document]
Comment (kimwnasptd) [reply components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/authz.py]: Done.

**Comment by kimwnasptd** [comment]
@elikatsis thank you for the review. I've incorporated your comments and also splitted the commit.
PTAL

**Pull request merged**